tests/Sam/PhpFpmRuntimeTest.php

```diff
@@ -84,10 +84,9 @@ public function test warnings are logged()
     {
         $response = $this->invoke('/?warning=1');
 
-        self::assertSame(500, $response->getStatusCode(), $this->logs);
-        // Unfortunately with the temporary fix in https://github.com/mnapoli/bref/pull/216
-        // we must settle for an empty 500 response for now
-//        self::assertEquals('Hello world!', $this->getBody($response), $this->logs);
```

Comment:
This is good: that means that a PHP warning will no longer return a blank 500 page (that was our previous workaround).

PHP now behaves just as usual.